buildSrc/src/main/java/com/uber/okbuck/composer/android/AndroidLibraryRuleComposer.java

```diff
@@ -83,6 +83,7 @@ public static Rule compose(
       androidRule.language("kotlin");
     }
 
+    androidRule.isLintTarget(true);
```

Comment:
whats the use of this flag?

Reply:
this is to skip generating any lint arguments for android test targets since it depends on android rule target.